### PR TITLE
Updated typo

### PR DESCRIPTION
**Pull request by NyashaNziramasanga**

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 13.6s | ⚠️ +829ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.405 | 2.367 | -0.04 |
| / avg req/sec | 1039.55 | 1056.13 | +16.58 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.482 | 1.441 | -0.04 |
| /error-in-render avg req/sec | 1687.1 | 1734.92 | +47.82 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 938 B | 938 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.9s | 14.2s | ⚠️ +220ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 2a82a9a7f36c9da814ef9934a2db6fd57d1eb62e

File: examples/blog-starter-typescript/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# A statically generated blog example using Next.js and Markdown and TypeScript
+# A statically generated blog example using Next.js, Markdown and TypeScript
```
Comment (Timer) [inline examples/blog-starter-typescript/README.md:1]:
```suggestion
# A statically generated blog example using Next.js, Markdown, and TypeScript
```

**Review by Timer — APPROVED** [review]
Thanks!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.8s | 12.9s | ⚠️ +131ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.105 | 2.044 | -0.06 |
| / avg req/sec | 1187.76 | 1223.1 | +35.34 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.225 | 1.287 | ⚠️ +0.06 |
| /error-in-render avg req/sec | 2040.99 | 1942.43 | ⚠️ -98.56 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 938 B | 938 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.6s | 13.8s | ⚠️ +208ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 3b2c68dd656550b7a77c869b3c95de49109de107

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13s | 12.7s | -282ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.184 | 2.132 | -0.05 |
| / avg req/sec | 1144.84 | 1172.72 | +27.88 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.453 | 1.545 | ⚠️ +0.09 |
| /error-in-render avg req/sec | 1720.81 | 1617.8 | ⚠️ -103.01 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 946 B | 946 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 938 B | 938 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.7s | 14.3s | ⚠️ +588ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..7a20.js gzip | 9.85 kB | 9.85 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-d8a3f1c..06ea.js gzip | 6.72 kB | 6.72 kB | ✓ |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 87.3 kB | 87.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.12 kB | 6.12 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-e63c4e5..dule.js gzip | 5.79 kB | 5.79 kB | ✓ |
| polyfills-05..dule.js gzip | 30.8 kB | 30.8 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 82.6 kB | 82.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8e69b..8695.js gzip | 3.41 kB | 3.41 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-ddd176e..5566.js gzip | 1.29 kB | 1.29 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.66 kB | 7.66 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-663e6..dule.js gzip | 2.24 kB | 2.24 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-69bc264..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.29 kB | 5.29 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 275 B | 275 B | ✓ |
| _buildManife..dule.js gzip | 281 B | 281 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | NyashaNziramasanga/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 024926f8cb6e9562bdb1974dd977f0850180a605

**Review by timneutkens — APPROVED** [review]

**Pull request merged**